content/source/docs/enterprise-beta/api/modules.html.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: enterprise2
+page_title: "Modules - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-modules"
+---
+
+# Registry Modules API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## List modules
+
+Lists all the modules for a given organization. **Note** that all other endpoints have the subpath `/api/v2`, this endpoint does not use that subpath.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /api/registry/v1/modules/:organization |
+
+
+### Parameters
+- `:organization` (`string: <required>`) - Specifies the organization for which to list the modules
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  https://atlas.hashicorp.com/api/registry/v1/modules/skierkowski-v2
+```
+
+### Sample Response
+
+```json
+{
+  "meta": {
+    "limit": 15,
+    "current_offset": 0
+  },
```

Comment:
Should we cross-link to Public API docs here since this pagination format and behaviour is not the same as other TFE JSON API endpoints but is documented in public API?

Reply:
Yeah, good call. I'll add more context around it and link to those docs.